### PR TITLE
RD-3250 Update node to v16

### PR DESCRIPTION
**Pull request by kubama**
## Description
Updated node to v16

## Screenshots / Videos
N/A

## Checklist

- [X] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I verified that all tests and checks have passed.
- [X] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [X] I added proper labels to this PR.

## Tests
https://jenkins.cloudify.co/blue/organizations/jenkins/Stage-UI-System-Test/detail/Stage-UI-System-Test/3357/pipeline

 

## Documentation
Updated README

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -95,7 +95,7 @@
         "sequelize-cli": "^6.4.1"
     },
     "engines": {
-        "node": "14.x"
+        "node": "16.x"
```
Comment (qooban) [inline backend/package.json:98]: I see change in `backend/package-lock.json` file, but no dependencies change in `backend/package.json`. Is it intentional or unplanned change?

File: jenkins/build-pod.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ spec:
         cpu: 0.3
         memory: 256Mi
   - name: node
-    image: cypress/browsers:node14.15.0-chrome96-ff94
+    image: cypress/browsers:node16.14.0-chrome99-ff97
```
Comment (qooban) [inline jenkins/build-pod.yaml:25]: I'm wondering, maybe we should move to the newer Chrome/FF by using `node16.14.2-slim-chrome103-ff102` image? 🤔

File: package.json [hunk 3]
```diff
@@ -50,7 +50,7 @@
     },
     "dependencies": {
         "@dnd-kit/core": "^4.0.0",
-        "@dnd-kit/modifiers": "^3.0.0",
+        "@dnd-kit/modifiers": "^4.0.0",
```
[the remark this answers is not in the document]
Comment (kubama) [reply package.json:53]: Yes, it exactly covers https://cloudifysource.atlassian.net/browse/RD-5328

File: backend/package.json (hunk 1 above)
Comment (kubama) [reply backend/package.json:98]: It's intentional and related to npm upgrade

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (qooban) [reply package.json:53]: @Vorbert-Kruk As you are working on RD-5328, I'm mentioning you here. You can possibly consult with @kubama as he did some related changes already in this PR.

File: jenkins/Jenkinsfile [hunk 4]
```diff
@@ -35,14 +35,33 @@ pipeline {
...
+          dir("${STAGE_DIR}/conf") {
+            withVault([configuration: configuration, vaultSecrets: secrets]) {
+              sh '''
+                jq ".maps.accessToken = \\"$MAPS_ACCESS_TOKEN\\"" app.json > app.json.tmp && mv app.json.tmp app.json
+              '''
+            }
+          }
+          echo "Fix directory ownership"
+          dir("${STAGE_DIR}") {
+            sh "chown -R `whoami` ."
```
[the remark this answers is not in the document]
Comment (kubama) [reply jenkins/Jenkinsfile:61]: Done

File: jenkins/build-pod.yaml (hunk 2 above)
Comment (kubama) [reply jenkins/build-pod.yaml]: Looks like it's working just fine - system tests passed after such change :+1:.

**Review by qooban — APPROVED** [review]

**Pull request merged**